src/components/badges/BadgeBlock.tsx

```diff
@@ -35,12 +35,11 @@ function Badge({
       if (!account) throw new Error('No account found')
       if (!proof?.result) throw new Error('No proof found')
       const transaction = await WalletStore.mintDerivative(proof)
-      const mintedBadge =
-        GoerliContractsStore.connectedAccounts[account].applyTransaction(
-          transaction
-        )
+      if (onMinted) onMinted()
```

Comment:
We don't use it anywhere else, so I moved it higher so we can get the effect right after the proof card changes into derivative card (if we'll shoot this on L:43, then the confetti will shoot with a delay, like the proof deletion does)